### PR TITLE
[cookbook] Use Swift Mailer in BackBee

### PR DESCRIPTION
**Pull request by ndufreche**
backbee/backbee-standard#226

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ pages:
     - 'Cookbooks':
         - 'Create new commands': 'developper-documentation/cookbooks/create_new_commands.md'
         - 'Customize error pages': 'developper-documentation/cookbooks/customize_error_pages.md'
+        - 'Use Swift Mailer in BackBee': 'developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md'
```
Comment (mickaelandrieu) [inline mkdocs.yml]: can you remove capitals please ?

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ pages:
     - 'Cookbooks':
         - 'Create new commands': 'developper-documentation/cookbooks/create_new_commands.md'
         - 'Customize error pages': 'developper-documentation/cookbooks/customize_error_pages.md'
+        - 'Use Swift Mailer in BackBee': 'developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md'
```
Comment (mickaelandrieu) [inline README.md]: can you remove capitals please ?

File: doc/developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+namespace BackBee\Bundle\DocumentationBundle\Listener;
+
+use BackBee\Event\Event;
+use BackBee\Site\Site;
+
+class MailerListener
+{
+    public static function onSomethingNeedToBeMailed(Event $event)
+    {
+        // retrieve your current Applcation
```
Comment (mickaelandrieu) [inline doc/developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md]: Application

File: doc/developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+
```
Comment (mickaelandrieu) [inline doc/developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md]: can you please add headers ?

```markdown
# CookBooks

## Use Swiftmailer in BackBee

```

Comment (ndufreche) [inline doc/developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md]: Yes

Comment (ndufreche) [inline doc/developper-documentation/cookbooks/use_Swift_Mailer_in_BackBee.md]: It's done.

**Comment by mickaelandrieu** [comment]
ok, you can merge I will ask for a complete review to Joseph asap.

Thank you @ndufreche

**Pull request merged**